src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java

```diff
@@ -452,4 +452,41 @@ private GoalFormDto setupGoalFormDto() {
         result.setDeadline(LocalDate.of(2918, 1, 1));
         return result;
     }
+
+    @Test
+    @DisplayName("test setStatusAchieved: changes status")
+    void test_SetStatusAchieved() {
```

Comment:
No need to put the underscore right after "test" - `testSetStatusAchieved` is OK.